monitoring.yml

```diff
@@ -150,7 +153,7 @@ properties:
       - database_type: "influxdb"
         database_name: (( meta.influxdb.database ))
         url: (( "http://" jobs.influxdb.networks.monitoring.static_ips.[0] ":8086" ))
-        name: "influxdb_production"
+        name: (( meta.grafana.name ))
```

Comment:
That would make sense--I was just looking down a few lines and saw `meta.grafana.influxdb_user` and `meta.grafana.influxdb_password`, which is why it turned out like this. Think we can get rid of `meta.grafana.influxdb_*` and keep the influxdb properties in `meta.influxdb`?